### ISSUE TITLE
Video get stuck but sound is good

### ISSUE THREAD

**Issue by leo108** [videojs/video.js#5664]
## Description
I'm using the newest version of videojs with VHS to play HLS videos, and find that image got stuck but the sound keeps playing sometimes, and try to replay the video it may work well or still broken, it happens to small videos in most cases.

I have checked the network panel in Chrome's console and make sure that .ts files are loaded correctlly.

I think it's a bug since 6.x version with videojs-contrib-hls, paste the m38u url (https://cdn.dubb.com/videos/2/1543758817/360p.m3u8) to [the videojs-contrib-hls demo page](https://videojs.github.io/videojs-contrib-hls/) can reproduce this bug too.

## Steps to reproduce

You can check the bug reproduce here: https://leo.dubb.com/v/GkFicm

jsfiddle url : http://jsfiddle.net/ndu7kaoq/

video m3u8 url: https://cdn.dubb.com/videos/2/1543758817/360p.m3u8

### Expected
Images should play well.

### Actual
Images got stuck sometimes.

### Error output
no error in conosle

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs

7.40

#### browsers

Chrome 71.0.3578.80

#### OSes

maxOS Mojave

### plugins

no plugin

**Comment by leo108** [comment]
I found a workaround to this bug after hours of testing, just reset player's src after the video ended.

```
player.on('ended', () => {
  // in order not to autoplay after reseting the video source
  player.autoplay(false);
  player.src(player.currentSource());
});
```

**Comment by leo108** [comment]
but I found this will make the play break in mobile after video ended

**Comment by clichedmoog** [comment]
Is this issue fixed?

**Comment by motionarray** [comment]
We're currently having this same issue. Anyone find a fix?

**Comment by gkatsev** [comment]
I can reproduce this. One thing I noticed is that it's only an issue with replays. Also, trying to do a replace manually by seeking to 0 and calling play reproduced the behavior but seeking to 0.1 and calling play works every time.

**Comment by motionarray** [comment]
Hmm. This is happening to us whenever we refresh the page...

**Comment by leo108** [comment]
I don't think this issue should be closed, the newest version v7.5.5 is still buggy and even worse than v6.x with the `videojs-contrib-hls` plugin.

**Comment by gkatsev** [comment]
Looks like this is fixed in VHS 1.x branch. It's working as good as VLC is for me. A release will be happening soon.
